copy/src/App.jsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from "react";
+import { Card } from "./component/Card/card.jsx";
+import { Header } from "./component/Header/header.jsx";
+import { Footer } from "./component/Footer/footer.jsx";
+import { SearchBar } from "./component/SearchBar/searchbar.jsx";
+import defaultImage from "./card-component-default.png";
+import "./style.css";
+
+const getFetch = async () => {
+  const response = await fetch(
+    "https://bootcamp-api.codeit.kr/api/sample/folder"
+  );
+  const body = await response.json();
+  return body;
+};
+
+const getUserData = async () => {
+  const response = await fetch(
+    "https://bootcamp-api.codeit.kr/api/sample/user"
+  );
+  const body = await response.json();
+  return body;
+};
+
+export default function App() {
+  const [folder, setFolder] = useState(null);
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = await getFetch();
+      setFolder(data.folder);
+
+      const user = await getUserData();
+      setUser(user);
+    };
+    getData();
+  }, []);
+
+  return (
+    <>
+      {folder !== null ? (
+        <Header folder={folder} user={user !== null ? user : ""} />
+      ) : undefined}
```

Comment:
```suggestion
      {folder !== null && (
        <Header folder={folder} user={user ?? ""} />
      )}
```
[중요도: 중간] 굳이 삼항연산자로 undefined를 할 필요 없이 &&연산자로 조건에 충족할 때만 return 시키는 것도 방법입니다.
[중요도: 중간] user가 없을 경우를 상정하게 된다면 ??연산자를 써보시는 것도 좋아요.